dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css

```diff
@@ -3732,6 +3726,14 @@ html, body {
   box-shadow: 0;
 }
 
+.listingTable table.dijitComboButtonFocused {
```

Comment:
needed because this rule: https://github.com/dotCMS/core/blob/2dd33fa42efeb7de9bf425f274db7acd793cdd3a/dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css#L7214